website/source/docs/platform/k8s/service-sync.html.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: "docs"
+page_title: "Service Sync - Kubernetes"
+sidebar_current: "docs-platform-k8s-service-sync"
+description: |-
+  The services in Kubernetes and Consul can be automatically synced so that Kubernetes services are available to Consul agents and services in Consul can be available as first-class Kubernetes services.
+---
+
+# Syncing Kubernetes and Consul Services
+
+The services in Kubernetes and Consul can be automatically synced so that Kubernetes
+services are available to Consul agents and services in Consul can be available
+as first-class Kubernetes services. This functionality is provided by the
+[consul-k8s project](https://github.com/hashicorp/consul-k8s) and can be
+automatically installed and configured using the
+[Consul Helm chart](/docs/platform/k8s/helm.html).
+
+**Why sync Kubernetes services to Consul?** Kubernetes services synced to the
+Consul catalog enable Kubernetes services to be accessed by non-Kubernetes
+nodes that are part of the Consul cluster or by other distinct Kubernetes
+clusters. For non-Kubernetes nodes, they can access services using the standard
+[Consul DNS](/docs/agent/dns.html) or HTTP API.
+
+**Why sync Consul services to Kubernetes?** Syncing Consul services to
+Kubernetes services enables non-Kubernetes services (such as external to
+the cluster) to be accessed in a native Kubernetes way: using kube-dns,
+environment variables, etc. This makes it very easy to automate external
+service discovery, including hosted services like databases.
+
+## Installation and Configuration
+
+The service sync is done using an external long-running process in the
+[consul-k8s project](https://github.com/hashicorp/consul-k8s). This process
+can run both in or out of a Kubernetes cluster. However, running this within
```

Comment:
"This process can run _both_ in or out of a Kubernetes cluster."

Should there be one instance of the process running at a time, or does it require two versions (one internal to kubernetes and one external)? It's not entirely clear from the wording.

If only one is needed, changing `both` -> `either` might be more clear.